manifests/base/management-clusters-fleet-collections.yaml

```
@@ -17,3 +19,21 @@ spec:
     - namespace: kube-system
       server: https://kubernetes.default.svc
     - server: https://kubernetes.default.svc
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: management-clusters-fleet-collections
```

```suggestion
  name: management-clusters-fleet-collection
```

Or maybe even:

```suggestion
  name: management-clusters-fleet-provider-collection
```

Just in case we decide to deploy this `shared` collection.